core/src/main/java/com/microsoft/applicationinsights/internal/util/SSLUtil.java

```diff
@@ -3,10 +3,18 @@
 import java.io.File;
 import javax.net.ssl.SSLHandshakeException;
 
+import com.microsoft.applicationinsights.customExceptions.FriendlyException;
 import com.microsoft.applicationinsights.internal.config.connection.ConnectionString.Defaults;
 
 public class SSLUtil {
 
+    public static void throwSSLFriendlyException(String url) {
+        boolean isUsingCustomKeyStore = (System.getProperty("javax.net.ssl.trustStore") != null);
+        throw new FriendlyException(SSLUtil.getSSLFriendlyExceptionBanner(url),
+                SSLUtil.getSSLFriendlyExceptionMessage(),
+                SSLUtil.getSSLFriendlyExceptionAction(url, isUsingCustomKeyStore),
+                SSLUtil.getSSLFriendlyExceptionNote());
```

Comment:
now can make the internal `get...` methods private (or package protected if needed for tests), and no need to prefix with `SSLUtil.`, just normal method calls

Reply:
also, can we log custom key store location since we have it?
```
return "Please import the SSL certificate from " + completeUrl + ", into your custom java key store (" + customStorePath + ").\n" +
                    "Learn more about importing the certificate here: https://go.microsoft.com/fwlink/?linkid=2151450";
```

Reply:
you are right. Probably I shouldnt make changes EOD, my brain stops working ;)